### PR TITLE
docs(readme): update with Usage section

### PR DESCRIPTION
**Pull request by rohanmurthy**
Documentation on using the [data loader](https://github.com/godaddy/split-javascript-data-loader/blob/master/src/load-data.js#L5).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,53 @@ Install via npm:
...
+
+### Function
+
+#### loadDataIntoLocalStorage
+
+You can use this function to store the following in browser `localStorage`:
+- Freshness of your data
+- Serialized split data
+- Count of splits using segments
+- Indicators for which segments a `userId` is a part of
```
Comment (celiawaggoner) [inline README.md]: i'm still confused about this, what's `userId` here? the segment data that the serializer returns is not specific to an individual GoDaddy customer. it's just segments in use by splits and the keys (the values in `added` that are part of those segments)

File: README.md [hunk 2]
```diff
@@ -10,6 +10,53 @@ Install via npm:
...
+Please note that this package is intended to be used in tandem with a split data serializer to create the `serializedData` parameter properly.
+An example of such serializer is [`split-node-serializer`](https://github.com/godaddy/split-node-serializer).
+
+```js
+import loadDataIntoLocalStorage from '@godaddy/split-data-loader'
+
+loadDataIntoLocalStorage({
+  serializedData: {
+    segmentsData: {
+      "segment_1": ['test-visitor-1', 'test-visitor-2', 'test-shopper-1'],
```
Comment (celiawaggoner) [inline README.md]: this will actually look like:
`"segment_1": "{ name: 'segment_1', added: ['test-visitor-1', 'test-visitor-2', 'test-shopper-1']}"`

File: README.md (hunk 1 above)
Comment (celiawaggoner) [reply README.md]: i'd just call this `serialized segment data`

Comment (rohanmurthy) [reply README.md]: The `userId` is either the hashed shopperId or visitorGuid value that pertains to the shopper/visitor. The `userId` is the "key" that would be a value in the `added` array.

Comment (celiawaggoner) [reply README.md]: whoops, i was misreading this. how it is makes sense to me

File: README.md [hunk 3]
```diff
@@ -10,6 +10,53 @@ Install via npm:
...
+    splitsData: {
+      "experiment_1": "{ name: 'experiment_1', status: 'foo' }",
+      "experiment_2": "{ name: 'experiment_2', status: 'bar' }"
+    },
+    usingSegmentsCount: 2
+  },
+  userId: 'test-visitor-1'
+})
+```
+
```
Comment (celiawaggoner) [inline README.md]: include an example of what `window` would look like after this function is called?

File: src/load-data.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 const TILL_KEY = 'SPLITIO.splits.till'
 
-export default function loadDataIntoLocalStorage ({ serializedData = {}, userId = '' }, windowLocalStorage = {}) {
+export default function loadDataIntoLocalStorage ({ serializedData = {}, userId = '' }, windowLocalStorage = window.localStorage) {
```
Comment (celiawaggoner) [inline src/load-data.js]: why this change?

Comment (rohanmurthy) [reply src/load-data.js]: We are using dependency injection (of `windowLocalStorage` param) for testing purposes. We shouldn't require clients to have to pass in `windowLocalStorage: window.localStorage` param.
